app/controllers/api/v3/analytics/user_analytics_controller.rb

```diff
@@ -37,10 +39,8 @@ def show
     @diabetes_reports_data = Reports::ReportsFakeFacilityProgressService.new(@current_facility.name).diabetes_reports
 
     respond_to do |format|
-      if Flipper.enabled?(:new_progress_tab_v2, current_user) || Flipper.enabled?(:new_progress_tab_v2)
+      if new_progress_tab_enabled?
         format.html { render :show_v2 }
-      elsif Flipper.enabled?(:new_progress_tab_v1, current_user) || Flipper.enabled?(:new_progress_tab_v1)
-        format.html { render :show_v1 }
       else
         format.html { render :show }
```

Comment:
Show "Update required" instead of old progress tab

Reply:
Show the **_old progress tab_** when `new_progress_tab` is disabled (In Ethiopia and Srilanka)
Show **_"Update Required"_** when `new_progress_tab` is enabled and `app version < min_app_version`
Show the **_new progress tab_** when `new_progress_tab` is enabled and `app version >= min_app_version`